apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/extract/s3/DataSetManifest.java

```diff
@@ -66,12 +75,12 @@ public final class DataSetManifest implements Comparable<DataSetManifest> {
   /**
    * Constructs a new {@link DataSetManifest} instance.
    *
-   * @param timestampText the value to use for {@link #getTimestampText()}
-   * @param sequenceId the value to use for {@link #getSequenceId()}
-   * @param syntheticData the value to use for {@link #isSyntheticData()}
```

Comment:
Without the link the `getTimestampText()` etc would not survive refactoring.  These are generally pretty useless anyway.  I recommend changing the text to read "value for field of same name.`

Reply:
I moved this comment to the field declaration